README.md

```diff
@@ -20,28 +20,32 @@ yarn add build-docs
 ## Usage
 
 ```js
-const buildDocs = require('build-docs');
+const buildDocs = require('./');
```

Comment:
Ahh, to generate this example bit i run `node example.js` and i must've forgotten to edit that bit.